styles/Vocab/Tarantool/accept.txt

```diff
@@ -33,4 +33,15 @@ storages
 dereference
 upsert
 tt
-SEQSCAN
\ No newline at end of file
+SEQSCAN
+iproto
+autoincrement
+etcd
+xlog
```

Comment:
Not sure we should encourage using "xlog" as a word in plain text. The correct entity name is "write-ahead log".